domainservice/src/testFixtures/kotlin/com/example/delivery/OrderFixture.kt

```diff
@@ -96,4 +101,28 @@ object OrderFixture {
     )
 
     val updatedDomainOrder = domainOrder.copy(shipmentDetails = randomUpdateDomainShipmentDetails)
+
+    val productNotFoundException = ProductNotFoundException("Product not found")
+    val userNotFoundException = UserNotFoundException("User not found")
+    val orderNotFoundException = OrderNotFoundException("Order not found")
+    val unexpectedException = NullPointerException()
```

Comment:
IMO it is overengineering, you can write same code in-place

but in your pet-project it is acceptable